app/assets/javascripts/discourse/app/routes/user-private-messages-warnings.js

```diff
@@ -0,0 +1,7 @@
+import createPMRoute from "discourse/routes/build-private-messages-route";
+
+export default createPMRoute(
+  "warnings",
```

Comment:
Might be worth throwing this into a constant since it's also used in `user-private-messages.js`.